src/Stratis.Bitcoin/BlockPulling2/BlockPullerBehavior.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Interfaces;
+using Stratis.Bitcoin.P2P.Peer;
+using Stratis.Bitcoin.P2P.Protocol;
+using Stratis.Bitcoin.P2P.Protocol.Behaviors;
+using Stratis.Bitcoin.P2P.Protocol.Payloads;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.BlockPulling2
+{
+    /// <summary>
+    /// Relation of the node's puller to a network peer node.
+    /// Keeps all peer-related values that <see cref="BlockPuller"/> needs to know about a peer.
+    /// </summary>
+    public interface IBlockPullerBehavior : INetworkPeerBehavior
+    {
+        /// <summary>Relative quality score of a peer.</summary>
+        /// <remarks>It's a value from <see cref="MinQualityScore"/> to <see cref="MaxQualityScore"/>.</remarks>
```

Comment:
Add the type `BlockPullerBehavior.MinQualityScore` on both

Reply:
Fixed